addon/package.json

```diff
@@ -2,7 +2,7 @@
   "title": "Test Pilot",
   "name": "testpilot-addon",
   "id": "@testpilot-addon",
-  "version": "2.0.2",
+  "version": "2.0.3",
```

Comment:
No, you only change it in package.json. The webpack copies that version over to install.rdf and manifest.json